packages/@okta/okta-auth-js/test/spec/factor-enroll.js

```diff
@@ -1,6 +1,6 @@
 jest.mock('cross-fetch');
 
-var util = require('../util/util'),
+var util = require('@test/support/util'),
```

Comment:
nit: I much prefer having each variable individually declared so we don't have to skim more than the relevant line, which is also consistent with the rest of the repo.